src/assets/voices/zh.json

```diff
@@ -0,0 +1,47 @@
+{
+    "category": [
+        {
+            "name": "moe",
+            "title": "心脏病警告，萌音注意！！！",
+            "voices": [
+                {
+                    "name": "ohayou",
+                    "path": "ohayou.mp3",
+                    "title": "Good Morning, Mornin~ heheh"
```

Comment:
can you help to translate this too? it's button text btw.. thank you :pray:

Reply:
Most of them are onomatopoeia, so I can only try to translate them. #1